### PR TITLE
Exec only to permitted users fix

### PR DESCRIPTION
**Pull request by mayhuberman**

### CODE REVIEW

File: apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java [hunk 1]
```diff
@@ -43,10 +39,10 @@ public AuthenticationFilter(AuthenticationService authenticationService) {
     }
 
     @Override
-    public void init(FilterConfig filterConfig) throws ServletException {}
```
Comment (roiravhon) [inline apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java]: Why?

File: apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java [hunk 2]
```diff
@@ -43,10 +39,10 @@ public AuthenticationFilter(AuthenticationService authenticationService) {
...
+    public void init(FilterConfig filterConfig) {}
 
     @Override
-    public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain) throws IOException, ServletException {
```
Comment (roiravhon) [inline apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java]: Why?

File: apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java (hunk 1 above)
Comment (mayhuberman) [reply apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java]: Unused, unnecessary, it was grey

File: apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java (hunk 2 above)
Comment (mayhuberman) [reply apollo-backend/src/main/java/io/logz/apollo/websockets/exec/AuthenticationFilter.java]: same

**Pull request merged**